core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/keystore/service/KeyStoreData.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2010, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Check the other license headers as well.

Reply:
Though this file looks like newly added, it is an old file with a git history.